connect/runtime/src/main/java/org/apache/kafka/connect/runtime/AbstractHerder.java

```diff
@@ -750,4 +751,38 @@ private String trace(Throwable t) {
         return keys;
     }
 
+    @Override
```

Comment:
(Left thoughts in `ConnectorPluginsResource` about possible ways we could refactor this to reduce the workload on the REST layer to handle some of the classloading logic)